src/java.desktop/windows/native/libawt/windows/awtmsg.h

```diff
@@ -209,23 +209,16 @@ enum {
     WM_AWT_WINDOW_SETACTIVE,
     WM_AWT_LIST_SETMULTISELECT,
     WM_AWT_HANDLE_EVENT,
-    WM_AWT_PRINT_COMPONENT,
     WM_AWT_RESHAPE_COMPONENT,
     WM_AWT_SETALWAYSONTOP,
     WM_AWT_BEGIN_VALIDATE,
     WM_AWT_END_VALIDATE,
     WM_AWT_FORWARD_CHAR,
-    WM_AWT_FORWARD_BYTE,
-    WM_AWT_SET_SCROLL_INFO,
     WM_AWT_CREATECONTEXT,
     WM_AWT_DESTROYCONTEXT,
     WM_AWT_ASSOCIATECONTEXT,
     WM_AWT_GET_DEFAULT_IME_HANDLER,
     WM_AWT_HANDLE_NATIVE_IME_EVENT,
-    WM_AWT_PRE_KEYDOWN,
-    WM_AWT_PRE_KEYUP,
-    WM_AWT_PRE_SYSKEYDOWN,
-    WM_AWT_PRE_SYSKEYUP,
 
     /* deleted DND mesg's */
 
```

Comment:
I guess this comment can safely be removed. What does it add? There had been some messages that were removed a long time ago.